dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilder.java

```diff
@@ -337,19 +337,20 @@ private static DependencyGraph levelOrder(
   }
 
   /**
-   * Returns true if {@code exceptionAndPath.getPath} does not contain {@code optional} dependency
-   * and the path does not contain {@code scope:provided} dependency.
+   * Returns true if {@code dependencyPath} does not contain {@code optional} dependency and the
+   * path does not contain {@code scope:provided} dependency.
    */
-  private static boolean isUnacceptableResolutionException(ExceptionAndPath exceptionAndPath) {
```

Comment:
Argument ExceptionAndPath is unnecessary. Only exceptionAndPath.getPath is needed.